plugins/org.jkiss.dbeaver.data.gis.view/src/org/jkiss/dbeaver/ui/gis/GeometryDataUtils.java

```diff
@@ -124,14 +141,18 @@ public static void setGeometryProperties(@NotNull IResultSetController controlle
     }
 
     @NotNull
-    public static RGB makeGeometryColor(int index) {
+    private static RGB makeGeometryColor(int index) {
         if (index == 0) {
             return Display.getCurrent().getSystemColor(SWT.COLOR_BLUE).getRGB();
         } else {
             return UIColors.getColor(index).getRGB();
         }
     }
 
+    private static String rgbToHex(RGB rgb) {
```

Comment:
We tend to specify `@NotNull` and `@Nullable` annotations throughout the codebase to simplify debugging and minimize risks of NPEs thanks to IDE inspections. Even if a method is private, it'd still require the programmer to read through it to figure out which arguments are nullable and whether it itself might return a null or not.

Reply:
Perhaps we could agree that everything should be non-nullable by default and only use @NotNull where necessary. What do you think?

Reply:
Unfortunately, this can't be enforced by default. Perhaps if we migrated to another library, such as the Checker Framework, it would be possible.